dali/operators/generic/erase/erase.cc

```diff
@@ -27,106 +27,133 @@
 namespace dali {
 
 DALI_SCHEMA(Erase)
-  .DocStr(R"code(Erases one or multiple regions from the image.
+  .DocStr(R"code(Erases one or more regions from the input tensors.
 
-The region is specified by an *anchor* (starting point) and a *shape* (dimensions).
+The region is specified by an ``anchor`` (starting point) and a ``shape`` (dimensions).
 Only the relevant dimensions are specified.
-Non-specified dimensions are treated as if the whole range of the axis was provided.
-To specify multiple regions, *anchor* and *shape* represent multiple points consecutively
-(e.g. *anchor* = (y0, x0, y1, x1, ...) and *shape* = (h0, w0, h1, w1, ...)).
-The arguments anchor and shape are interpreted according to the value of the argument *axis_names*
-(or alternatively the value of the argument *axes*). If no *axis_names*/*axes* arguments are provided,
-all the dimensions except 'C' (channels) must be specified.
+Not specified dimensions are treated as if the entire range of the axis was provided.
+To specify multiple regions, ``anchor`` and ``shape`` represent multiple points consecutively
+(for example, ``anchor`` = (y0, x0, y1, x1, ...) and ``shape`` = (h0, w0, h1, w1, ...)).
+The ``anchor`` and ``shape`` arguments are interpreted based on the value of the ``axis_names``
+argument, or, alternatively, the value of the ``axes`` argument. If no ``axis_names`` or
+``axes`` arguments are provided, all dimensions except ``C`` (channels) must be specified.
 
 **Example 1:**
 
-*anchor* = (10, 20), *shape* = (190, 200), *axis_names* = "HW", *fill_value* = 0
+``anchor`` = (10, 20), ``shape`` = (190, 200), ``axis_names`` = "HW", ``fill_value`` = 0
 
-input: *layout* = "HWC", *shape* = (300, 300, 3)
+input: ``layout`` = "HWC", ``shape`` = (300, 300, 3)
 
-The erase region covers the range from 10 to 200 in the vertical dimension (heigth) and goes from
-20 to 220 in the horizontal dimension (width). The range for the channel dimension goes from 0 to 3, as it
-was not specified. That is::
+The erase region covers the range between 10 and 200 in the vertical dimension (height)
```

Comment:
```suggestion
The erased region covers the range between 10 and 200 in the vertical dimension (height)
```
Is that even true? If it's shape, shouldn't it be `10 and 210` ? @klecki ?